src/structures/Guild.js

```diff
@@ -1192,7 +1192,9 @@ class Guild extends Base {
    * The data needed for updating a channel's position.
    * @typedef {Object} ChannelPosition
    * @property {ChannelResolvable} channel Channel to update
-   * @property {number} position New position for the channel
+   * @property {number} [position] New position for the channel
+   * @property {?CategoryChannel|Snowflake} [parent] Parent channel
```

Comment:
The reason why is explained in #6021

```suggestion
   * @property {?(CategoryChannel|Snowflake)} [parent] Parent channel
```

Reply:
At this point, this is more of an issue to track that this plan technically exists, rather than anything that's happening any time soon, so this will very likely be completely refactored, if implemented. Accordingly not going to merge any suggestions.